docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,209 @@
+.. _external_storage:
+
+External Storage in Renku Sessions
+==================================
+
+Renku supports accessing data from external storage services like
+S3, Azure Blob, WebDAV, etc. in interactive sessions. Access to external 
+storage can be configured via project settings. 
+
+In the sections below we describe how to add and manage storage configuration to your projects
+and highlight some current limitations:
+
+.. contents::
+  :local: 
+
+.. note::
+
+  Note that this feature is optional; it is enabled at
+  `renkulab.io <http://renkulab.io>`_  but other Renku deployments might not
+  support it. Contact your administrator for more details.
+
+.. warning::
```

Comment:
I think an additional warning should be written here about Read/Write access:
Read/Write access may expose you data to data loss, data corruption or data being overwritten. Consult the documentation of you cloud storage provider to learn how to setup access controls and data versioning if you want to protect your data.